src/Components/Announcment/Announcment.scss

```diff
@@ -0,0 +1,24 @@
+.Announcment {
+    background-color:whitesmoke;
+    margin: 30px 0;
+    border-top-style: groove;
+    display: block;
+
+    &__title {
+        font-size: 30px;   
+    }
+    
+    &__img {
+        width: 700px;
+        height: 300px;
+        border: 10px solid;   
+    }
+}
+
+.Announcment__img :hover {
+    width: 100px;
+    background-color: rgb(61, 59, 59);
+    transition: background-color .9s;
+    
+}
```

Comment:
should be under 

    &__img {
        width: 700px;
        height: 300px;
        border: 10px solid;   
    }